atd-vzd/migrations/default/1715960019005_crashes_list_and_fatality_metrics_views/up.sql

```diff
@@ -222,9 +222,15 @@ select
     lookups.collsn_lkp.label as collsn_desc,
     geocode_status.is_manual_geocode,
     geocode_status.has_no_cris_coordinates,
+    to_char(
+        public.crashes.crash_timestamp at time zone 'US/Central', 'YYY:MM:DD'
+    ) as crash_date_ct,
+    to_char(
+        public.crashes.crash_timestamp at time zone 'US/Central', 'HH24:MI:SS'
+    ) as crash_time_ct,
```

Comment:
xavier mentioned wanting these, and we're going to want to expose these fields in crash exports to replicate old functionality, so i went ahead and created them here and will add them to the crashes export in a follow up when I take on https://github.com/cityofaustin/atd-data-tech/issues/17944

Reply:
sounds/looks perfect!